- Работа с Ads (add, update, delete)

skat2006 opened the pull request:
- Реализовано добавление Ads с картинкой, изменение Ads и картинки, удаление Ads с картинкой и комментариями
- Увеличено покрытие логами
- в AdsController предварительно расставлены права на методы

avs-repo committed:
- Реализовано добавление Ads с картинкой, изменение Ads и картинки, удаление Ads с картинкой и комментариями
- Увеличено покрытие логами
- в AdsControllere предварительно расставлены права на методы

MaraOD555 commented on src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java:
```diff
@@ -43,7 +43,7 @@ public boolean login(String userName, String password) {
-            log.info("Пользователь удачно вошел - {}", userName);
+            log.info("Пользователь успешно вошел - {}", userName);
         } else {
             log.warn("Ошибка входа. Не верный пароль пользователя {} ", userName);
```
просто пока увидела:) неверный слитно

MaraOD555 commented on src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java:
```diff
@@ -43,7 +43,7 @@ public boolean login(String userName, String password) {
         boolean isLoggedIn = encoder.matches(password, encryptedPasswordWithoutEncryptionType);
         if (isLoggedIn) {
-            log.info("Пользователь удачно вошел - {}", userName);
+            log.info("Пользователь успешно вошел - {}", userName);
```
удачно было круто:)))

avs-repo committed:
- Реализована работа с аватаром добавление, изменение, удаление (AvatarService)
- Исправлены в Entity привязки avatar_id и poster_id
- Добавлен метод updateComment
- Часть сервисов перестроены по CRUD

avs-repo committed:
- Орфография :))
